modules/statistics/statistics.js

```diff
@@ -16,10 +21,12 @@ var JitsiTrackError = require("../../JitsiTrackError");
 // downloading their API as soon as possible and (2) do the downloading
 // asynchronously.
 function loadCallStatsAPI() {
-    ScriptUtil.loadScript(
-            'https://api.callstats.io/static/callstats.min.js',
-            /* async */ true,
-            /* prepend */ true);
+    if(!isCallstatsLoaded)
+        ScriptUtil.loadScript(
+                'https://api.callstats.io/static/callstats.min.js',
+                /* async */ true,
+                /* prepend */ true);
+    isCallstatsLoaded = true;
```

Comment:
I think {} could help here. There's no point in isCallstatsLoad = true; when the condition !isCallstatsLoaded is false.